.devcontainer/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.177.0/containers/javascript-node/.devcontainer/base.Dockerfile
+
+# [Choice] Node.js version: 16, 14, 12
+ARG VARIANT="16-buster"
```

Comment:
Do we support Node 16 for the dvc.org cc @rogermparent ?

Reply:
It works well.

Reply:
I'm pretty sure node 16 is what broke CI builds and that's why we now specify <=15 in package.json.

Reply:
I think I tried 14 first and it didn't work.